SAMtags.tex

```diff
@@ -234,13 +239,39 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
-
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+Barcode sequence (Identifying the sample/library), with any quality scores (optionally) stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
```

Comment:
This sentence talks about " unique molecular identifiers" in the barcode sequence tag description.  I think `unique molecular identifiers` -> `barcode sequences`.  Also, consider a read structure: R1: `4B2S4B100T` and R2: `10B100T`.  Should the tag be `NNNN-NNNN-NNNNNNNNNN` or `NNNNNNNNNN-NNNNNNNNNN`.  I.e. should we delimit the reads or the template barcode segments?